source/extensions/filters/http/ratelimit/ratelimit.h

```diff
@@ -114,16 +114,18 @@ class Filter : public Http::StreamFilter, public Filters::Common::RateLimit::Req
   void setEncoderFilterCallbacks(Http::StreamEncoderFilterCallbacks& callbacks) override;
 
   // RateLimit::RequestCallbacks
-  void complete(Filters::Common::RateLimit::LimitStatus status,
-                Http::HeaderMapPtr&& headers) override;
+  void complete(Filters::Common::RateLimit::LimitStatus status, Http::HeaderMapPtr&& headers,
```

Comment:
Internally, can we use consistent request/response naming?

Reply:
Used response_headers and request_headers_to_add everwhere.